spec/lib/word_counter_spec.rb

```diff
@@ -45,7 +45,7 @@
     expect(word_counter.count).to eq(2)
   end
 
-  %w(* ~ !? - ~* ~!).each do |char|
+  %w(* ~ !? - ~* ~! |+).each do |char|
```

Comment:
Why is it `|+` in this case, instead of just `|`?

Reply:
> The fix makes sense to me! Just a few more things:
> 
> * Can you fix the Hound/Rubocop warnings?
> * We'll need to reset the word count on every work for the update to apply. There's a task for that in lib/tasks/work_tasks.rake, but it requires a specific language. Can you update the task so that if a language argument is given, it should process works with that language, but if not, it should process all works?


Yeah both should be no problem, I'll fix it after exams. Wasn't sure if you all wanted the hounds things updated, since I didn't really introduce new code, but just followed the style that was already present.

Reply:
> Why is it `|+` in this case, instead of just `|`?

There's really no reason, just followed the previous examples. Now that I look at it again and think about it, I don't even understand why `!?` `~*` and `~!` are there since it goes by char anyway.

Reply:
> Yeah both should be no problem, I'll fix it after exams. Wasn't sure if you all wanted the hounds things updated, since I didn't really introduce new code, but just followed the style that was already present.

In general we fixed warnings on lines added/modified and leave the rest alone. No rush and good luck on your exams!